src/test/java/uk/gov/justice/digital/hmpps/keyworker/services/KeyworkerStatsBatchServiceTest.java

```diff
@@ -0,0 +1,121 @@
+package uk.gov.justice.digital.hmpps.keyworker.services;
```

Comment:
These are Java classes as the other tests are also Java, but maybe we should be using Kotlin for all new files?

Reply:
If possible use Kotlin. You may find that you need to delombok some of the code.